lib/java/ydoc-server/src/main/java/org/enso/ydoc/polyfill/web/Timers.java

```diff
@@ -92,7 +84,17 @@ public Object execute(Value... arguments) {
     };
   }
 
-  private Runnable execute(Value func, Object[] args) {
+  private Runnable runnable(Value func, Object[] args) {
     return () -> executor.execute(() -> func.executeVoid(args));
```

Comment:
I guess this is wrong. It should be just `() -> func.executeVoid(args)`.

Reply:
I had concurrent execution errors in the JS code before. Maybe with a single executor, it won't be the case anymore. I will test it.

Reply:
When there were multiple threads, it was indeed essential to return back to the original thread before executing the JavaScript function. However, now, when there is just a _single thread scheduling executor_ everything can be much simpler.